db/migrate/20230331165525_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key:{to_table: :users}, index: { name: 'comments_authors' }
```

Comment:
- [ ] Good job working on this but please help make sure it references `:author` just like it is in the post migration so that it matches the ERD diagram specifics.